string itemName
uint8[] idArray
uint8 idCnt
float64[] setData
---
bool complete
